homeassistant/components/system_bridge/__init__.py

```diff
@@ -308,6 +309,7 @@ def device_info(self) -> DeviceInfo:
         return DeviceInfo(
             configuration_url=self._configuration_url,
             connections={(dr.CONNECTION_NETWORK_MAC, self._mac_address)},
+            identifiers={(DOMAIN, self._uuid)},
```

Comment:
Just a quick question: Why not move this whole device info into the above SystemBridgeEntity and remove the SystemBridgeDeviceEntity altogether?

Reply:
Good point 👍